psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="false"
+    errorLevel="5"
```

Comment:
any more as per other projects?

Reply:
Lower is better, and we can have lvl 4 with 0 modifications, so let's do that.